yjit.rb

```diff
@@ -217,7 +217,9 @@ def _print_stats
       $stderr.puts "invalidation_count:    " + ("%10d" % stats[:invalidation_count])
       $stderr.puts "constant_state_bumps:  " + ("%10d" % stats[:constant_state_bumps])
       $stderr.puts "inline_code_size:      " + ("%10d" % stats[:inline_code_size])
+      $stderr.puts "inline_code_gc_size:   " + ("%10d" % stats[:inline_code_gc_size])
       $stderr.puts "outlined_code_size:    " + ("%10d" % stats[:outlined_code_size])
+      $stderr.puts "outlined_code_gc_size: " + ("%10d" % stats[:outlined_code_gc_size])
```

Comment:
Here I think it would be better to name `inline_code_gc_size` => `inline_freed_bytes` to be more consistent with the variable name you use for the counter internally.